lib/datasource/docker/common.spec.ts

```diff
@@ -131,4 +132,171 @@ Object {
       expect(headers).toBeNull();
     });
   });
+  describe('isECRMaxResultsError', () => {
+    it('returns true when the error matches the maxResults error', () => {
+      const err = {
```

Comment:
```suggestion
      const err: RequestError = {
```

🤷‍♂️

Reply:
Reworked these tests to be tested through `getPkgReleases`.